invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -77,6 +77,17 @@
   <div class="ui relaxed grid">
     <div class="two column row top-padded">
       <div class="eleven wide column">
+        {% if record.access.record == "restricted" and record.access.embargo.active %}
+          <div class="ui warning message">
+            <div class="header"><i class="clock outline icon"></i> Embargoed </div>
+            <p>You are viewing an embargoed record. The record and files will be made publicly available on {{format_date(record.access.embargo.until)}}.</p>
```

Comment:
:globe_with_meridians:  We are marking the other piece of text below as translatable, why not mark this one as translatable too?

Reply:
Yes, should also be marked as translatable (I forgot ....), thanks!